src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.ts

```diff
@@ -95,8 +97,21 @@ export class ChatMessageEditor {
         this.isTextMode = isTextMode;
         if (isTextMode) {
             this.editorDiv.classList.add('text-mode');
+            this.animationOff();
         } else {
             this.editorDiv.classList.remove('text-mode');
+            this.animationOff();
+        }
+    }
+
+    private animationOff() : void {
```

Comment:
Are you sure it won't give an issue with updats due to blazor and DOM have different classes list?